### ISSUE TITLE
Partition Metric not rounding decimals

### ISSUE THREAD

**Issue by CosmicodeStudio** [laravel/nova-issues#3309]
- Laravel Version: 7.30.4
- Nova Version: 3.22.0
- PHP Version: 7.4.8
- Database Driver & Version: MySQL 5.7.31
- Operating System and Version: Ubuntu 18.04.5 LTS

### Description:

Partition Metrics are not rounding decimals in total only.

Here's an image showing the issue.
![Screenshot 2021-03-25 at 10 41 42](https://user-images.githubusercontent.com/34129939/112460393-bf3bc000-8d56-11eb-9e2f-ea9f00c1c238.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:

To test just create a new partition metric and use this data:
```
$result = [
    'Name 1' => 75.07,  
    'Name 2' => 48.09,
    'Name 3' => 16.39,
    'Name 4' => 15.02,
    'Name 5' => 13.86,
    'Name 6' => 7.22,
    'Name 7' => 5.93,
    'Name 8' => 5.62,
    'Name 9' => 2.72,
    'Name 10' => 2.21
]

return $this->result($result);
```

**Comment by crynobone** [comment]
Unable to replicate the issue:

![image](https://user-images.githubusercontent.com/172966/112564962-228d2700-8e17-11eb-9ebb-c6bfabd2d202.png)

You can review it from https://github.com/nova-issues/issue-3309/commit/a65f35f011879b90f357c9dcdd315b4818f3db1b

**Comment by CosmicodeStudio** [comment]
Sorry, I forgot to add that you need to add `$roundingPrecision = 2` to the partition metric.

**Comment by crynobone** [comment]
This has been fixed in 3.23.0 but please upgrade to 3.23.1. https://nova.laravel.com/releases
